packages/core/src/util/renderer.ts

```diff
@@ -260,6 +260,7 @@ export interface OwnPropsOfLabel extends OwnPropsOfRenderer {
 
 export interface OwnPropsOfEnum {
   options?: EnumOption[];
+  emptyLabel?: string;
```

Comment:
Hmm I don't think we should handle this via the props. For me this seems like more a decision on the renderer side whether they want to offer a `None` option.